server/src/main/java/org/opensearch/action/RoutingMissingException.java

```diff
@@ -46,6 +47,10 @@
 
     private final String id;
 
+    public RoutingMissingException(String index, String id) {
+        this(index, MapperService.SINGLE_MAPPING_NAME, id);
+    }
+
     public RoutingMissingException(String index, String type, String id) {
```

Comment:
This constructor shouldn't be used, right? May be better to make it private?

Reply:
It's still used in IndexRequest, Update, Get, and MultiGet API calls. I was planning on removing it with the last PR that no longer needs it.

Reply:
:+1: